pkg/operator/ceph/cluster/cluster.go

```diff
@@ -216,7 +216,14 @@ func (c *ClusterController) initializeCluster(cluster *cluster, clusterObj *ceph
 		opts := metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", k8sutil.AppAttr, mgr.AppName)}
 		mgrDeployments, err := c.context.Clientset.AppsV1().Deployments(cluster.Namespace).List(opts)
 		if err == nil && len(mgrDeployments.Items) > 0 && cluster.ClusterInfo != nil {
-			c.configureCephMonitoring(cluster, clusterInfo)
+			// Populate ClusterInfo with the last value
+			var err error
+			cluster.mons.ClusterInfo, _, _, err = mon.LoadClusterInfo(c.context, c.namespacedName.Namespace)
+			if err != nil {
+				return errors.Wrap(err, "failed to load local cluster info for configuring monitoring")
+			}
+
+			c.configureCephMonitoring(cluster, cephUser)
```

Comment:
Another fix needed from the merge conflict...
```suggestion
			c.configureCephMonitoring(cluster, cluster.ClusterInfo)
```